tools/osbuilder/rootfs-builder/cbl-mariner/rootfs_lib.sh

```diff
@@ -22,5 +22,69 @@ build_rootfs()
 	info "install packages for rootfs"
 	$DNF install ${EXTRA_PKGS} ${PACKAGES}
 
+	# Reduce the image size, for faster TEE memory measurement.
+	local MARINER_REMOVED_PACKAGES=( \
```

Comment:
If I remember correctly, Mitch was working on getting rid of REMOVED_PACKAGES.

Reply:
Right, we'll have to wait until his changes are merged to make the changes here.